project.godot

```diff
@@ -12,7 +12,7 @@ config_version=5
 
 config/name="mparty"
 run/main_scene="res://main/main.tscn"
-config/features=PackedStringArray("4.0", "GL Compatibility")
+config/features=PackedStringArray("4.1", "GL Compatibility")
```

Comment:
Why did this change? Is this intentional?

Reply:
sry wasnt intentional, ig my godot did it because mine automatically updated to 4.1 bcs linux